sdk/resourcemanager/automanage/armautomanage/README.md

```diff
@@ -49,9 +61,146 @@ options := arm.ClientOptions {
+configuration["Backup/PolicyName"] = "dailyBackupPolicy"
+configuration["Backup/TimeZone"] = "UTC"
+configuration["Backup/InstantRpRetentionRangeInDays"] = 2
+configuration["Backup/SchedulePolicy/ScheduleRunFrequency"] = "Daily"
+configuration["Backup/SchedulePolicy/ScheduleRunTimes"] = []string{"2022-07-27T12: 00: 00Z"}
+configuration["Backup/SchedulePolicy/SchedulePolicyType"] = "SimpleSchedulePolicy"
+configuration["Backup/RetentionPolicy/RetentionPolicyType"] = "LongTermRetentionPolicy"
+configuration["Backup/RetentionPolicy/DailySchedule/RetentionTimes"] = []string{"2022-07-27T12: 00: 00Z"}
+configuration["Backup/RetentionPolicy/DailySchedule/RetentionDuration/Count"] = 180
+configuration["Backup/RetentionPolicy/DailySchedule/RetentionDuration/DurationType"] = "Days"
+configuration["WindowsAdminCenter/Enable"] = false
+configuration["VMInsights/Enable"] = true
+configuration["AzureSecurityCenter/Enable"] = true
+configuration["UpdateManagement/Enable"] = true
+configuration["ChangeTrackingAndInventory/Enable"] = true
+configuration["GuestConfiguration/Enable"] = true
+configuration["AutomationAccount/Enable"] = true
+configuration["LogAnalytics/Enable"] = true
+configuration["BootDiagnostics/Enable"] = true
+
+properties := armautomanage.ConfigurationProfileProperties{
+    Configuration: configuration,
+}
+
+id := "/subscriptions/[sub ID]/resourceGroups/resourceGroupName/providers/Microsoft.Automanage/configurationProfiles/configurationProfileName"
+resourceType := "Microsoft.Automanage/configurationProfiles"
+location := "eastus"
+environment := "dev"
+configurationProfileName := "name"
+
+// tags may be omitted 
+tags := make(map[string]*string)
+tags["environment"] = &environment
+
+newProfile := armautomanage.ConfigurationProfile{
+    ID:         &id,
+    Name:       &configurationProfileName,
+    Type:       &resourceType,
+    Location:   &location,
+    Properties: &properties,
+    Tags:       tags,
+}
+
+configProfilesClient.CreateOrUpdate(context.Background(), configurationProfileName, "resourceGroupName", newProfile, nil)
+```
+
+
+## Get an Automanage Configuration Profile
+
+```go
+profile, _ := configProfilesClient.Get(context.Background(), "configurationProfileName", "resourceGroupName", nil)
+data, _ := json.MarshalIndent(profile, "", "   ")
+
+fmt.Println(string(data))
+```
+
+
+## Delete an Automanage Configuration Profile
+
+```go
+configProfilesClient.Delete(context.Background(), "resourceGroupName", "configurationProfileName", nil)
+```
+
+
+## Get an Automanage Profile Assignment
+
+```go
+assignment, _ := assignmentClient.Get(context.Background(), "resourceGroupName", "default", "vmName", nil)
+data, _ := json.MarshalIndent(assignment, "", "   ")
+fmt.Println(string(data))
+```
+
+
+## Create an Assignment between a VM and an Automanage Best Practices Production Configuration Profile
+
+```go
+vmId := "/subscriptions/[sub ID]/resourceGroups/resourceGroupName/providers/Microsoft.Compute/virtualMachines/vmName"
```

Comment:
Could you please consolidate the placeholder?

Reply:
Done